releasenotes/notes/2.0/box-66874e4aacdaeec1.yaml

```diff
@@ -0,0 +1,48 @@
+---
+features_circuits:
+  - |
+    A new control-flow op, :class:`.BoxOp`, its associated :meth:`.QuantumCircuit.box` method and context manager
```

Comment:
It looks like `BoxOp` is not in the api docs, so this link doesn't work. We should add it in this PR too.

Reply:
Added, but we should probably also add proper docs in the future @jakelishman

Reply:
Looking back at the PR, I think I did all the docs except the `qiskit.circuit` module ones - I updated the tables and whatnot in `QuantumCircuit`, just not `qiskit.circuit`.

Reply:
Well, let me know if you have any suggestions for the 4 lines I added here (it was really just to prompt the autosummary)

Reply:
Lets open a tracking issue to check over the circuit docs for 2.0.1. I had to add the `Duration` class to the circuit docs too, but I just stuffed an autoclass in there next to delay without any explanation.